flink-table/flink-table-runtime-blink/src/main/java/org/apache/flink/table/runtime/operators/TableStreamOperator.java

```diff
@@ -46,4 +47,13 @@ public void dispose() throws Exception {
 		super.dispose();
 	}
 
+	/**
+	 * Compute memory size from memory faction.
+	 */
+	public long computeMemorySize() {
+		double memFraction = getOperatorConfig().getManagedMemoryFractionOnHeap() +
+				getOperatorConfig().getManagedMemoryFractionOffHeap();
+		MemoryManager memManager = getContainingTask().getEnvironment().getMemoryManager();
+		return ((long) memManager.computeNumberOfPages(memFraction)) * memManager.getPageSize();
```

Comment:
you can use `MemoryManager#computeMemorySize` if you want the memory size directly.

Reply:
Good to know `MemoryManager#computeMemorySize` has to be introduced.